netcdf/src/file.rs

```diff
@@ -423,6 +423,17 @@ impl FileMut {
         let (ncid, name) = super::group::get_parent_ncid_and_stem(self.ncid(), name)?;
         super::variable::add_variable_from_identifiers(ncid, name, dims, T::NCTYPE)
     }
+
+    /// Flush pending buffers to disk to minimise data loss in case of termination.
```

Comment:
I don't really have an HPC background, but I have trouble imagining I could trust the contents of a file after a crash without some kind of logging scheme.

But this is also useful to detect I/O errors that happen when closing the file, since `drop` isn't going to return anything.

Reply:
That's some good points you're raising. This was taken from the [official documentation](https://docs.unidata.ucar.edu/netcdf-c/current/group__datasets.html#gaf2d184214ce7a55b0a50514749221245), and should cover crashes outside of `netCDF` (e.g. when computing data).

Reply:
https://danluu.com/file-consistency/ is worth a read by anyone feeling lucky enough to try to "minimize data loss" this way.